src/components/LibraryItem.tsx

```diff
@@ -258,6 +263,14 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
         }
     }
 
+    // Unexpand all other slibling items when one item is being expanded
```

Comment:
Wait, it should be named `onSingleChildItemWillExpand`, and not `onSingleChildItemExpanded`.